psa-crypto/Cargo.toml

```diff
@@ -19,5 +19,6 @@ log = "0.4.8"
 serde = { version = "1.0.110", features = ["derive"] }
 
 [features]
-default = ["with-mbed-crypto"]
+default = ["with-mbed-crypto", "no-std"]
 with-mbed-crypto = ["psa-crypto-sys"]
+no-std = []
```

Comment:
How would one compile this crate with only the `no-std` feature activated? Would it be like:
```
cargo build --no-default-features --features no-std
```

Reply:
Yes, precisely!